packages/types/src/userType.ts

```
@@ -7,6 +7,7 @@ export interface IUser {
   email: string;
   password: string;
   role: TRole;
+  teams: String[];
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)